### PR TITLE
fix: Upgrade elliptic to address security vulnerability

### PR DESCRIPTION
**Pull request by jennifer-shehane**
### Additional details

This addresses a critical vulnerability that was surfaced here: https://security.snyk.io/vuln/SNYK-JS-ELLIPTIC-7577916

This was in our `@cypress/webpack-batteries-included-preprocessor` package as a sub dependency of another dependency. Issue opened for that package here: https://github.com/browserify/crypto-browserify/issues/234

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Review by ryanthemanuel — APPROVED** [review]

File: npm/webpack-batteries-included-preprocessor/README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ module.exports = (on) => {
 }
 ```
 
-Other than the `typescript` option, this preprocessor supports the same options as [@cypress/webpack-preprocessor](https://github.com/cypress-io/cypress/tree/develop/npm/webpack-preprocessor#readme), so see its README for more information.
```
Comment (jennifer-shehane) [inline npm/webpack-batteries-included-preprocessor/README.md:47]: Just a file change to trigger the release in this package which is triggered by any file update in this package directory. This change is unrelated.

**Review by AtofStryker — APPROVED** [review]

**Comment by jennifer-shehane** [comment]
Snyk check is passing now.
